apps/balancer-tools/src/app/apr/(utils)/getFilteredApiUrl.ts

```diff
@@ -0,0 +1,27 @@
+import { SearchParams } from "../round/[roundId]/page";
+
+export const MIN_TVL = 1000;
+
+const convert = (key: string, value: string) => {
+  if (["sort", "order"].includes(key)) return value || undefined;
+  if (["minTVL", "maxTVL", "minAPR", "maxAPR"].includes(key))
+    return Number(value) || undefined;
+  if (["tokens", "type", "network"].includes(key))
+    return value ? value.split(",") : undefined;
+  return value;
+};
+function getFilterDataFromParams(searchParams: SearchParams) {
+  return Object.fromEntries(
+    Object.entries(searchParams).map(([key, value]) => [
+      key,
+      convert(key, value) || (key === "minTVL" ? MIN_TVL : undefined),
+    ]),
+  );
+}
+export default function getFilteredApiUrl(searchParams: SearchParams) {
+  const filteredData = getFilterDataFromParams(searchParams);
+  const params = Object.entries(filteredData)
+    .map(([key, value]) => (value !== undefined ? `&${key}=${value}` : ""))
+    .join("");
+  return `${process.env.NEXT_PUBLIC_SITE_URL}/apr/api/?&limit=10${params}`;
```

Comment:
`#would` add limit=10 as a default searchParam instead of hardcoding it to the URL

Reply:
`#should` use BASE_URL from `src/app/apr/api/route.ts`